src/Hooks.php

```diff
@@ -86,6 +86,24 @@ public static function onBeforeDataUpdateComplete( Store $store, SemanticData $s
 		Services::getDataAnnotator()->addAnnotations( $topic, $semanticData );
 	}
 
```

Comment:
Nitpick: Please add a comment similar to the comment at `onBeforeDataUpdateComplete`.

```suggestion
/**
* Hook to process information after an update has been completed.
*
* @param Store $store
* @param SemanticData $semanticData
* @return void
* @link https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/docs/technical/hooks/hook.store.afterdataupdatecomplete.md
*/
```